cmd/vcfFilter/vcfFilter.go

```diff
@@ -92,21 +92,27 @@ func usage() {
 
 // criteria by which vcf records are filtered.
 type criteria struct {
-	chrom                    string
-	groupFile                string
-	minPos                   int
-	maxPos                   int
-	minQual                  float64
-	ref                      string
-	alt                      []string
-	biAllelicOnly            bool
-	substitutionsOnly        bool
-	segregatingSitesOnly     bool
-	removeNoAncestor         bool
-	onlyPolarizableAncestors bool
-	formatExp                string
-	infoExp                  string
-	includeMissingInfo       bool
+	chrom                          string
+	groupFile                      string
+	minPos                         int
+	maxPos                         int
+	minQual                        float64
+	ref                            string
+	alt                            []string
+	biAllelicOnly                  bool
+	substitutionsOnly              bool
+	segregatingSitesOnly           bool
+	removeNoAncestor               bool
+	onlyPolarizableAncestors       bool
+	weakToStrongOrStrongToWeakOnly bool
+	noWeakToStrongOrStrongToWeak   bool
```

Comment:
Are these redundant to the lines below (refWeakAltStrongOnly, refStrongAltWeakOnly, notRefWeakAltStrong, notRefStrongAltWeak)?